test/wat2wasm/call_indirect.t

```diff
@@ -0,0 +1,51 @@
+  $ owi wat2wasm call_indirect.wat
+  $ owi run call_indirect.wasm --debug
+  typechecking ...
+  linking      ...
+  interpreting ...
+  stack        : [  ]
+  running instr: i32.const 0
+  stack        : [ i32.const 0 ]
+  running instr: i32.const 0
+  stack        : [ i32.const 0 ; i32.const 0 ]
+  running instr: i32.const 1
+  stack        : [ i32.const 1 ; i32.const 0 ; i32.const 0 ]
+  running instr: table.init 0 0
+  stack        : [  ]
+  running instr: elem.drop 0
+  stack        : [  ]
+  stack        : [  ]
+  running instr: call 1
+  calling func : func anonymous
+  stack        : [  ]
+  running instr: i64.const 1
+  stack        : [ i64.const 1 ]
+  running instr: i32.const 0
+  stack        : [ i32.const 0 ; i64.const 1 ]
+  running instr: call_indirect 126 
+  owi: internal error, uncaught exception:
+       Not_found
```

Comment:
This should be fixed by adding @chambart's part, right?

Reply:
I hope so ;)

Reply:
Same with new tests : `wat2wasm symbolic` and `wat2wasm concolic`
https://github.com/OCamlPro/owi/commit/f3a359c3af76098333b2e8c38f619e830ec3f745#diff-7ff2e62bd9bf02aa0e66d6d93d6546fbaefabe80b6bd1f87d7972d0706135c84

Reply:
> This should be fixed by adding @chambart's part, right?

OK, done by @chambart ;)

Reply:
> Same with new tests : `wat2wasm symbolic` and `wat2wasm concolic`

done https://github.com/OCamlPro/owi/pull/232/commits/67047505a2f923ddbe242d64b6922c640281eaf5